packages/react/src/components/select/custom/custom-option.tsx

```diff
@@ -23,9 +24,10 @@ export function CustomOption({
   useEffect(() => initialize(children, optionValue), []);
 
   const selectOption = () => disabled || select(children, optionValue);
-
   const selected = value == optionValue;
 
+  if (hidden) return null;
```

Comment:
[Minor]

```suggestion
  if (hidden) return null;

  const selected = value == optionValue;
```

Because no point to check for `selected` when you expect to return `null`.

Reply:
Right. I hoisted it above both `const` declarations.